wall_e_models/models.py

```diff
@@ -45,35 +50,39 @@ def insert_records(cls, records: List[BanRecord]) -> None:
     @sync_to_async
     def insert_record(cls, record: BanRecord) -> None:
         """Adds entry to BanRecord table"""
-        record.save()
+        try:
+            record.save()
+        except Exception:
```

Comment:
```suggestion
        except Exception:
        except IntegrityError:
```

broad Exception clauses are generally a bad ide